test/parseDetectorTables.test.ts

```diff
@@ -0,0 +1,26 @@
+const vsCodeIntegration: string = 'Latest time seen by detector. To be used in VSCode integration.';
+suite('Detector Dataset Parser', () => {
```

Comment:
I like to name the file and suite the same as the function it's testing. This is a good example: https://github.com/microsoft/vscode-azuretools/blob/master/appservice/test/convertStacksToPicks.test.ts#L472

FWIW I like `parseDetectorTables` better than `tableUtils`